DUI3-DX/Converters/Rhino/Speckle.Converters.Rhino7/ToHost/Raw/FlatPointListRawToHostConversion.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Rhino.Collections;
+using Speckle.Converters.Common.Objects;
+using Speckle.Core.Logging;
+
+namespace Speckle.Converters.Rhino7.ToHost.Raw;
+
+public class FlatPointListRawToHostConversion : IRawConversion<IList<double>, Point3dList>
```

Comment:
This is more of a utility function, but it is a conversion of sorts so it matched the pattern and does work when injecting it just fine.

Not sure how this would play out on the new registration you're looking at.